### ISSUE TITLE
Removed from CRAN?

### ISSUE THREAD

**Issue by gwerbin** [gaborcsardi/falsy#6]
It looks like this was removed from CRAN earlier this year:

> Archived on 2016-05-02 as check problems were not corrected.

Do you know what those problems were? I like the idea behind this package and I'd be happy to help out if you'd like.

**Comment by gaborcsardi** [comment]
Yes, sorry. `falsy` is kind of dangerous (See https://github.com/gaborcsardi/falsy#warnings), and since no other packages depended on it, I took it down.

I found it that I usually just define `"%||%" <- function(l, r) if (is.null(l)) r else l` nowadays, i.e. I only treat `NULL` as falsy.

If you really need it, I can put it back on CRAN, though.

**Comment by gwerbin** [comment]
Fair enough. However your other package Seer apparently does depend on it (according to its DESCRIPTION)

**Comment by gaborcsardi** [comment]
Good point. I "fixed" seer, to install `falsy` form GitHub automatically.

**Comment by maelle** [comment]
Should the README state more clearly that it was archived from CRAN and why? Currently, the CRAN downloads badge gives a wrong impression until you click on it. :-)

**Comment by gaborcsardi** [comment]
I'll just archive this, once crandb does not need it.
